backend/model/document_keywords_generator.rb

```diff
@@ -78,18 +93,33 @@ def self.format_identifier(record)
     },
 
     :record_identifier => proc {|record|
-      Array(record['linked_records']).map {|linked|
-        label = case linked['_resolved']['jsonmodel_type']
-            when 'resource'
-              :resource_identifier_keyword
-            when 'accession'
-              :accession_identifier_keyword
-        end
-
-        if label
-          Keyword.new(label, DocumentKeywordsGenerator.format_identifier(linked['_resolved']))
-        end
-      }.compact
+      if record['jsonmodel_type'] == ('event' || 'archival_object')
```

Comment:
Hi @jdshaw,

The `||` on the right will always evaluate to `'event'`, so the `archival_object` case will never happen.  It's tricky because it reads well in English, but doesn't quite do what you mean :)

You could instead do:

     if ['event', 'archival_object'].include?(record['jsonmodel_type'])